eslint-config-novoda/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "eslint-config-novoda",
+  "version": "1.0.0",
+  "description": "ESLint default configuration for Novoda packages",
+  "main": "index.js",
+  "keywords": [
+    "eslint",
+    "eslintconfig",
+    "novoda",
+    "config"
+  ],
+  "contributors": [{
+    "name": "Francesco Pontillo",
+    "email": "francesco@novoda.com"
+  }],
+  "homepage": "https://github.com/novoda/eslint-config-novoda",
```

Comment:
Yeah, but you can update the link once it's moved :P Anyways, I won't leave this block the PR 😂